File: src/00/sigma_core/src/ontology_gen/00/dir8Y.rs

pub fn dir8Y(n: i32) -> i32 {
    if n == 2 || n == 4 || n == 5 {
        -1
    } else if n == 3 || n == 6 || n == 7 {
        1
    } else {
        0
    }
}

File: src/_as/00/dir8_y.ts
// SSoT: src/ontology/spatial/dir8_y.md

@inline
export function dir8_y(n: i32): i32 {
if (n == 2 || n == 4 || n == 5) return -1;
if (n == 3 || n == 6 || n == 7) return 1;
return 0;
}
